docs/UserGuide.adoc

```diff
@@ -72,6 +72,11 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Listing all persons : `sort`
+
+Shows a list of all persons, along with their non-private details, in the address book. +
```

Comment:
This is exactly the same as `list` command, how is `sort` different from `list` then?